boefjes/boefjes/config.py

```diff
@@ -2,43 +2,58 @@
 from pathlib import Path
 from typing import Optional
 
-from pydantic import AnyHttpUrl, BaseSettings, PostgresDsn
+from pydantic import AmqpDsn, AnyHttpUrl, BaseSettings, Field, IPvAnyAddress, PostgresDsn
 
 
 class RuntimeConfiguration(Enum):
     LOCAL = "local"
 
 
 class Settings(BaseSettings):
-    base_dir: Path = Path(__file__).parent.resolve()
-    log_cfg: Path = Path(__file__).parent / "logging.json"
+    base_dir: Path = Field(Path(__file__).parent.resolve())
```

Comment:
base_dir is not really a setting. It also only used in the tests, so we should use a normal variable for that.

Reply:
I'll look into refactoring it for all modules.